optaplanner-persistence/optaplanner-persistence-jaxb/src/test/java/org/optaplanner/persistence/jaxb/api/score/AbstractScoreJaxbXmlAdapterTest.java

```diff
@@ -103,22 +106,25 @@
             throw new IllegalStateException("Marshalling or unmarshalling for input (" + input + ") failed.", e);
         }
         assertEquals(expectedScore, output.getScore());
+
+        String newLine = System.lineSeparator();
```

Comment:
Using System.lineSeperator() always makes me cringe. Can't remember/motivate why though...

Reply:
I haven't found anything wrong about it. Even OpenJDK seems to be switching to it:
https://bugs.openjdk.java.net/browse/JDK-8198645